dynamic_programming/maximum-sum-increasing-subsequence.py

```diff
@@ -0,0 +1,32 @@
+#############################
+# Author: Ujjal Das( github.com/ujjaldas132)
+# File: lis.py
+# comments:Given an array A . Fin d the sum of maximum sum increasing subsequence of the given array.
+#           if A=[1 101 2 3 100 4 5] then answer shold be 106>>>>>>1,2,3,100
+#############################
+
+def max_sum(a):
+    n=len(arr)
+    
+    sum_list=[a[0]]
+    max_val=a[0]
+    for i in range(1,n):
+        temp=a[i]
+        j=i-1
+        while j>-1:
+            if a[j]<a[i]:
+                if temp<a[i]+sum_list[j]:
+                    temp=a[i]+sum_list[j]
+                if ans[j]==t:
```

Comment:
flake8 is saying that __ans__ and __t__ need to be defined _before_ they are used.